### DISCUSSION TITLE
How do I get all the entities in physical space

### DISCUSSION THREAD

**Discussion by dannisliang** [bepu/bepuphysics1#43] [Q&A]
How do I get all the entities in physical space?

**Comment by Gavin-Williams** [comment]
I looked up the documentation https://github.com/bepu/bepuphysics1/blob/master/Documentation/GettingStarted.md and scrolled down just a little bit and found this..
`space.Add(new Box(new Vector3(0, 4, 0), 1, 1, 1, 1));`

**Comment by RossNordby** [comment]
You can enumerate entities with `Space.Entities`. Entities do not include things like static meshes, which (if I remember correctly) don't have a central storage anywhere. They just get added to the `BroadPhase` which isn't required to maintain a contiguous list. You'd have to track them yourself.
